index.js

```diff
@@ -54,4 +55,12 @@ module.exports = (robot) => {
       await checkMergeConflictsModule.checkMergeConflictsInAllPullRequests(context);
     }
   });
+
+
+  robot.on('schedule.repository', async context => {
+    if (whitelistedAccounts.includes(context.repo().owner.toLowerCase())) {
+      console.log('Periodic Check...');

```

Comment:
Add a log which is more explanatory -- i.e explain what the periodic check is exactly doing?

Reply:
Please add this as a comment then.

Reply:
Done.